### PR TITLE
Add viasat.apis.docker and docker-query command.

### PR DESCRIPTION
**Pull request by kanaka**
This library/command queries the standard Docker Hub API. The command takes two parameters: namespace and image. For Docker official packages, use "library" as the namespace (this is default for general docker commands but is required for this comand).

It does not currently support authentication (for private repos and higher rate limits).

Also, improve the base-url defaulting for viasat.apis.npm.

### CODE REVIEW

File: docker-query.cljs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  --fields FIELDS                     Fields to print (comma or space separated)
+  --sort FIELD                        Sort using FIELD
+  --rsort FIELD                       Reverse sort using FIELD
+  --field-max FIELD_MAX               Maximum field width (when not --verbose) [default: 100]
+  --no-headers                        Skip printing headers
+  --json                              Print full results as JSON (default)
+  --edn                               Print full results as EDN
+  --pretty                            Pretty print --json or --edn
+
+  --base-url URL                      NPM base URL [env: NPM_BASE_URL]
```
Comment (jonsmock) [inline docker-query.cljs]: NPM vs. Docker Hub

File: docker-query.cljs [hunk 2]
```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env nbb
+
+(ns artifactory-docker-query
```
Comment (jonsmock) [inline docker-query.cljs]: Maybe just `docker-query`?

**Review by jonsmock — APPROVED** [review]
Approving - only some cosmetic changes requested inline.

**Comment by kanaka** [comment]
@jonsmock good catches, thanks! I've applied those changes and merged.

**Pull request merged**